test/sanity/code-smell/update-bundled.py

```diff
@@ -48,7 +48,7 @@ def get_bundled_libs(paths):
     bundled_libs = set()
     for filename in fnmatch.filter(paths, 'lib/ansible/compat/*/__init__.py'):
         bundled_libs.add(filename)
-    for filename in fnmatch.filter(paths, 'lib/ansible/utils/_antsibull_docs_parser/*.py'):
+    for filename in fnmatch.filter(paths, 'lib/ansible/_vendor/antsibull_docs_parser/*.py'):
```

Comment:
As part of the PR to update the sanity test, we can change this to just `lib/ansible/_vendor/`.